pnpm-workspace.yaml

```diff
@@ -1,5 +1,10 @@
 packages:
   - docs
-  - packages/*
+  - packages/**
```

Comment:
this is intentional and in prep for the test suite PR which will be under `packages/web-fragments/test`